skimage/measure/tests/test_fit.py

```diff
@@ -361,7 +361,7 @@ def estimate(self, data):
             return True
 
         def residuals(self, data):
-            return 1.0
+            return np.ones_like(data, dtype=np.double)
```

Comment:
shouldn't this be `np.ones(len(data), dtype=np.double)`?

Reply:
ohhh, yeah you are right.